### DISCUSSION TITLE
My docker deployment tries to load a Vercel library (speed-insights/script.js)

### DISCUSSION THREAD

**Discussion by ilogus** [lobehub/lobehub#2010] [Self Hosting] [answered]
Hello, I open a question here because today I realize that my deployment of LobeChat v0.147.8 with docker is trying to load (unsuccessfully with a 404 error) a library vercel speed-insights/script.js

Is it possible that this is a bug in the application? Or have I misconfigured something? I don't see any link between docker and vercel.

![image](https://github.com/lobehub/lobe-chat/assets/10232690/38718190-8014-4b96-8ac3-03f21d24ef21)

**Comment by arvinxx** [comment, marked as answer]
Oh. Maybe it's because of this 

https://github.com/lobehub/lobe-chat/blob/main/src/app/layout.tsx#L29

I think maybe I can make it open only when deploy on vercel.

**Comment by ilogus** [comment]
If it's not a big job, it's a nice little feature 😄 

and just so I know, is this speed-insights feature for Vercel users or for your own statistics?

Thank !

**Reply by arvinxx** [reply]
vercel users

**Reply by ilogus** [reply]
Thanks for this info !
